src/util/sss_cli_cmd.c

```diff
@@ -105,9 +105,6 @@ const char *sss_cmd2str(enum sss_cli_command cmd)
         return "SSS_NSS_GETNETGRENT";
     case SSS_NSS_ENDNETGRENT:
         return "SSS_NSS_ENDNETGRENT";
-    /* SSS_NSS_INNETGR:
```

Comment:
You need to wipe other traces of this command handling as well then.

There is 
`sss_client/sss_cli.h:130:    /* SSS_NSS_INNETGR     = 0x0064, */`
but probably more.

Reply:
Done.